packages/atomic/cypress/integration/facets/category-facet/category-facet.cypress.ts

```diff
@@ -256,6 +256,8 @@ describe('Category Facet Test Suites', () => {
         function setupSelectFirstParent() {
           setupGoDeeperLastLevel();
           pressParentButton(0);
+          cy.wait(TestFixture.interceptAliases.Search);
+          cy.wait(TestFixture.interceptAliases.UA);
```

Comment:
If possible, I would instead assert something.

Reply:
I think the assertion would be `CategoryFacetAssertions.assertFocusActiveParent()`, which is already done after this... I couldn't really put my finger on what the problem was here, but a lot of the times the element wouldn't be focused for this assertion.. I'd suggest that we try to add these two lines, but happy to close this PR if you think we shouldn't do it! let me know